pdc/apps/messaging/backends/dummy.py

```diff
@@ -15,4 +17,5 @@
 class DummyMessenger(BaseMessenger):
 
     def send_message(self, topic, msg):
-        logger.info('Sending to %s:\n%s' % (topic, json.dumps(msg, sort_keys=True, indent=2)))
+        if settings.DEBUG:
+            logger.info('Sending to %s:\n%s' % (topic, json.dumps(msg, sort_keys=True, indent=2)))
```

Comment:
What about instead changing the level to debug and changing the configuration

```
if DEBUG:
    LOGGING['loggers']['pdc.apps.messaging.backends.dummy'] = {'level': 'DEBUG'}
```

Reply:
Added debug logging for everything if `DEBUG == True`. Though debug log level shouldn't be ever enabled on prod and stage.